tools/misc/browse_dataset.py

```diff
@@ -80,6 +80,7 @@ def main():
     cfg = Config.fromfile(args.config)
     if args.cfg_options is not None:
         cfg.merge_from_dict(args.cfg_options)
+    file_client = mmengine.FileClient(**cfg.file_client_args)
```

Comment:
We may need to check if `file_client_args` exists in the config.

Reply:
Done.